src/appleseed/renderer/kernel/intersection/triangletree.cpp

```diff
@@ -1420,6 +1422,10 @@ bool TriangleLeafVisitor::visit(
                 m_shading_point.m_ray.m_tmax = t;
                 m_shading_point.m_bary[0] = static_cast<float>(u);
                 m_shading_point.m_bary[1] = static_cast<float>(v);
+                m_shading_point.m_offset_function = [](const ShadingPoint& shading_point)
```

Comment:
Can we bind `refine()` directly, without a lambda?

Reply:
What's the cost of binding a function to `m_offset_function`? This is a performance-critical piece of code.